ai_flow/model/internal/events.py

```diff
@@ -53,23 +55,24 @@ class EventContextConstant(object):
     TASK_EXECUTION_ID = 'task_execution_id'
     WORKFLOW_SCHEDULE_ID = 'workflow_schedule_id'
     WORKFLOW_SNAPSHOT_ID = 'workflow_snapshot_id'
+    SEQUENCE_NUMBER = 'sequence_number'
+    TASK_STATUS = 'task_status'
 
 
 class TaskStatusChangedEventKey(EventKey):
     """TaskStatusChangedEventKey represents an event of the task status changed."""
     def __init__(self,
+                 namespace: str,
                  workflow_name: str,
-                 task_name: str,
-                 namespace: str = DEFAULT_NAMESPACE):
+                 task_name: str,):
         """
         :param workflow_name: The name of the workflow to which the task belongs.
         :param task_name: The task's name.
-        :param namespace: The task's namespace.
         """
-        super().__init__(namespace=namespace,
-                         name=workflow_name,
+        super().__init__(namespace=None,
+                         name='{}.{}.{}'.format(namespace, workflow_name, task_name),
```

Comment:
If we use `'{}.{}.{}'.format(namespace, workflow_name, task_name)` as name, how does the event receiver extract the namespace & workflow_name & task_name if any of them contains "dot" ?